packages/markdown-it-container/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sage-bionetworks/markdown-it-container",
-  "version": "4.0.1",
+  "version": "0.0.0",
```

Comment:
Set all packages to v0.0.0 since the version is set in the workflow.  The engineer releasing the new version should reference NPM to identify the previous release version.